README.md

```diff
@@ -23,6 +23,9 @@ This helm chart installs and configures the following operators:
 
 ## Helm Chart Installer
 
+> [!IMPORTANT]
+> It is heavily recommended you run the Helm Chart on a fresh cluster that does not have pre-existing GitOps/ArgoCD, Pipelines/Tekton and Developer Hub Operators.
```

Comment:
```suggestion
> It is recommended you run the Helm chart on a cluster that does not have a pre-existing GitOps/ArgoCD, Pipelines/Tekton and Developer Hub operators.
```

Reply:
I will remove heavily but I think grammatically `Chart` and `Operators` should be capitalized, no? Since they are in reference to an actual titled thing?

Reply:
Actually I think just Operator is capitalized